docs/source/_static/hackrtd.css

```diff
@@ -0,0 +1,112 @@
+/* Temporary hack to work around bug in rtd theme 2.0 through 2.4
+   See https://github.com/rtfd/sphinx_rtd_theme/pull/382
+*/
+pre {
+    line-height: normal !important;
+}
+
+/* Make .. deprecation:: blocks visible
+ * (by default they're entirely unstyled)
+ */
+.deprecated {
+    background-color: #ffe13b;
+}
+
+/* Add a snakey triskelion ornament to <hr>
```

Comment:
This does not appear to work, https://trio.readthedocs.io/en/stable/ has it on the top <hr /> but https://pytest-trio--112.org.readthedocs.build/en/112/ does not

Reply:
Thanks, I see I forgot to copy over `ornament.svg`.  But also, could you share a screenshot of where you see it?  I've looked up what a triskelion is `:]` and checked the contents of the file but I can't find it present at the Trio link.  For example, should I be seeing it here?

![image](https://user-images.githubusercontent.com/543719/96120877-340ff180-0ebd-11eb-87c7-74d49ec85f83.png)